flocker/volume/test/filesystemtests.py

```diff
@@ -618,7 +620,11 @@ def got_volumes(copied):
                     with from_filesystem.reader() as reader:
                         with to_filesystem.writer() as writer:
                             data = reader.read()
-                            writer.write(data[1:])
+                            try:
+                                writer.write(data[1:])
+                            except IOError as e:
+                                if e.errno != errno.EPIPE:
```

Comment:
I understand the situation but as long as the test (and the API that it tests) remains in the current form, and I think that that could be for a while, I would like the test to test the right thing.